src/legacy/core_plugins/kibana/public/discover/np_ready/angular/helpers/_add_to_siri.ts

```diff
@@ -17,22 +17,23 @@
  * under the License.
  */
 
-export function addToSiri(series, point, id, yLabel, yFormat, zFormat, zLabel) {
+import { Point } from './_get_point';
+
+export interface Serie {
+  id: string;
+  values: Point[];
+}
+
+export function addToSiri(series: Map<string, Serie>, point: Point, id: string) {
   id = id == null ? '' : id + '';
 
   if (series.has(id)) {
-    series.get(id).values.push(point);
+    (series.get(id) as Serie).values.push(point);
     return;
   }
 
   series.set(id, {
-    id: id.split('-').pop(),
-    rawId: id,
-    label: yLabel == null ? id : yLabel,
-    count: 0,
+    id: id.split('-').pop() as string,
     values: [point],
```

Comment:
In the [component ](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kibana/public/discover/np_ready/angular/directives/histogram.tsx#L215)the 'values' is used only, so I removed all other field.